docs/en/installation.md

```diff
@@ -1,56 +1,24 @@
 # Installation of the RealMe module
 
-The module is best installed via Composer by running the below command:
+The module is best installed via Composer, by adding the below to your composer.json. For now, we need to specify a 
+custom version of the excellent onelogin/php-saml module to fix some XMLDSig validation errors with the RealMe XML 
```

Comment:
Unfortunately yes. I haven't had a chance to properly fix this yet, and the upstream module author wasn't able to solve this without more time from me.